api/default.nix

```diff
@@ -41,6 +41,12 @@ with pkgs; rec {
       gzip
       gnugrep
       util-linux
+      nodejs-16_x
+      yarn
+      python3
+      gcc
+      gnumake
+      gnused
```

Comment:
I want to refrain from putting this all in the main env if possible - maybe we can only do this if we are building a dev container?

Reply:
I thought about that, but I didn't want to make develeopment and production packages different, what do you think?